src/server/grpc/p2pool.rs

```diff
@@ -77,6 +78,8 @@ where S: ShareChain
     stats_max_difficulty_since_last_success: Arc<RwLock<Difficulty>>,
     consensus_manager: ConsensusManager,
     submit_block_semaphore: Arc<Semaphore>,
+    tribe: Tribe,
```

Comment:
Let's change this to Squad and replace all instances of `tribe` with `Squad`

Reply:
done